pkg/sentry/socket/netfilter/netfilter.go

```diff
@@ -34,16 +34,33 @@ import (
 // shouldn't be reached - an error has occurred if we fall through to one.
 const errorTargetName = "ERROR"
 
-// metadata is opaque to netstack. It holds data that we need to translate
-// between Linux's and netstack's iptables representations.
-// TODO(gvisor.dev/issue/170): This might be removable.
+const (
+	matcherNameUDP = "udp"
+)
```

Comment:
I'm going to leave them separate for now, as there will soon be other matcher names (e.g. matcherNameTCP") added to the list. Let me know if you feel strongly otherwise.